ote/src/clj/ote/integration/import/gtfs.clj

```diff
@@ -183,17 +178,14 @@
       (log/info "Generating finnish regions and envelope for package " package-id)
       (gtfs-set-package-geometry db {:package-id package-id})
 
-      ;; IF handling was ok, remove all errors from the interface table
-      (specql/update! db ::t-service/external-interface-description
-                      {::t-service/gtfs-db-error nil ::t-service/gtfs-import-error nil}
-                      {::t-service/id interface-id})
-
       (catch Exception e
         (.printStackTrace e)
-        (specql/update! db ::t-service/external-interface-description
-                        {::t-service/gtfs-imported (java.sql.Timestamp. (System/currentTimeMillis))
-                         ::t-service/gtfs-db-error (str (.getName (class e)) ": " (.getMessage e))}
-                        {::t-service/id interface-id})
+        (specql/insert! db ::t-service/interface-download
```

Comment:
Would it be more robust to do the `log/warn` first and only then instert to db? Just in case there's a problem with the insertion.

Reply:
brilliant